### PR TITLE
CumulusCI onboarding

### PR DESCRIPTION
**Pull request by davidmreed**
Onboard project into CumulusCI

- Test metadata moves into `unpackaged`.
- Customizations to automation to cope with multiple package directories treated as a single package.
- Remove legacy material.

### CODE REVIEW

File: orgs/beta.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "orgName": "Declarative Lookup Rollup Summaries Tool - Beta Test Org",
```
Comment (Szandor72) [inline orgs/beta.json:2]: Using this config and the one below would result in a German org for me, depending which dev hub I use. 
Should we add 
`country` and `language` properties to help international Devs to end up with an English org setup - no matter which dev hub is used?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# DLRS
```
Comment (Szandor72) [inline README.md:1]: Would you know which readme.md wins? We have two readmes if we merge this. `readme.markdown` already exists.

File: orgs/beta.json (hunk 1 above)
Comment (davidmreed) [reply orgs/beta.json:2]: We certainly can. Is DLRS language-sensitive? I'm not sure what the impact would be of having development done in a non-English org; would that muck up the CustomObjectTranslations for any new schema?

File: README.md (hunk 2 above)
Comment (davidmreed) [reply README.md:1]: Oops. I will fix this.

File: orgs/beta.json (hunk 1 above)
Comment (Szandor72) [reply orgs/beta.json:2]: Yes, as far as I can remember this is exactly what happens.

**Review by Szandor72 — APPROVED** [review]
Thanks for correcting the readme. I will address language and other dev experience related features in a new branch.

**Pull request merged**